src/Result/ServerInfo.php

```diff
@@ -32,5 +32,8 @@ public function getSupportedPaymentMethods(): array
         return $this->getData()['supportedPaymentMethods'];
     }
 
-    // TODO add "syncStatus" structure
+    public function getSyncStatus(): ServerSyncStatus
+    {
+        return new ServerSyncStatus($this->getData()['syncStatus'][0]);
```

Comment:
I think we need a ServerSyncStatusList class like we have on other places where an array is returned. This endpoint will return the status for each shitcoin (XMR, LTC, etc) that might be configured. Not that I like it but if BTC is not returned first you won't see its status :D

Reply:
ah ok that makes a lot more sense, i thought greenfield just did something weird but turns out i forgot about shitcoins...